docs/configuration/settings.rst

```diff
@@ -447,7 +447,131 @@ Company settings
   :alt: Screenshot showing Company Merge Settings Configuration in Mautic
 
 * **Merge by unique fields with operator** - You can determine which operator to use when merging fields if there is more than one unique identifier.
-  
+
+Queue settings
+****************
+
+Purpose of the queuing
+======================
+
+Mautic can optionally use a queuing mechanism for sending emails. This feature is essential when Mautic is run **at large scale**. There will be more tasks that can utilize queuing in the future.
+
+When you enable queuing, emails are no longer sent immediately (e.g. within HTTP requests)
+but put in a queue and sent later using queue consumers (also known as workers). Using consumers help offload workload of your webserver and allows easier scaling of your instance.
+
+The queuing is disabled by default
+==================================
+
+After installing a fresh instance of Mautic, **the queuing feature is disabled** (the queue DSN is set to ``"sync://"``) as shown in the following screenshot.
+
+.. image:: images/queue-disabled.png
+  :width: 600
+  :alt: Queue is disabled
+
+How to enable the queuing
+=========================
+
+First you need to choose what technology (queuing transport) will drive your queue. The are several options available at the moment. It is up to you to choose which one fits your needs the best.
+
+**Currently available transports:**
+
+* :ref:`Doctrine`
+* :ref:`Redis`
+* :ref:`AMQP`
+* :ref:`Beanstalkd`
+* :ref:`Amazon SQS`
+
+Doctrine
+--------
+This transport is easy to setup as it does not require installing any additional extension.
+It uses database table ``messenger_messages`` for storing messages (the table name can be changed via options). The screenshot below shows the basic settings.
+
+.. image:: images/queue-doctrine.png
+  :width: 600
+  :alt: Example of Doctrine transport
+
+See `https://symfony.com/doc/5.4/messenger.html#doctrine-transport <https://symfony.com/doc/5.4/messenger.html#doctrine-transport>`_ for the complete list of configuration options.
```

Comment:
Please use the command `make link` in the docs folder, and properly create an x-ref here. Be sure to commit the link file too! Thanks!